### ISSUE TITLE
[Bug] unrecognized arguments error

### ISSUE THREAD

**Issue by Ishihara-Masabumi** [open-mmlab/mmaction2#2618]
### Branch

main branch (1.x version, such as `v1.0.0`, or `dev-1.x` branch)

### Prerequisite

- [X] I have searched [Issues](https://github.com/open-mmlab/mmaction2/issues) and [Discussions](https://github.com/open-mmlab/mmaction2/discussions) but cannot get the expected help.
- [X] I have read the [documentation](https://mmaction2.readthedocs.io/en/latest/) but cannot get the expected help.
- [X] The bug has not been fixed in the [latest version](https://github.com/open-mmlab/mmaction2).

### Environment

python 3.8
mmaction2 1.1.0

### Describe the bug

When I tried to run a training script, the following error occurred.

### Reproduces the problem - command or script

python tools/train.py configs/recognition/tin/tin_imagenet-pretrained-r50_8xb6-1x1x8-40e_sthv1-rgb.py     --work-dir work_dirs/tin_imagenet-pretrained-r50_8xb6-1x1x8-40e_sthv1-rgb randomness.seed=0 randomness.deterministic=True

### Reproduces the problem - error message

(openmmlab) dl@dl-machine:~/mmaction2/mmaction2$ python tools/train.py configs/recognition/tin/tin_imagenet-pretrained-r50_8xb6-1x1x8-40e_sthv1-rgb.py --work-dir work_dirs/tin_imagenet-pretrained-r50_8xb6-1x1x8-40e_sthv1-rgb randomness.seed=0 randomness.deterministic=True
usage: train.py [-h] [--work-dir WORK_DIR] [--resume [RESUME]] [--amp] [--no-validate] [--auto-scale-lr] [--seed SEED]
                [--diff-rank-seed] [--deterministic] [--cfg-options CFG_OPTIONS [CFG_OPTIONS ...]]
                [--launcher {none,pytorch,slurm,mpi}] [--local_rank LOCAL_RANK]
                config
train.py: error: unrecognized arguments: randomness.seed=0 randomness.deterministic=True

**Comment by Dai-Wenxun** [comment]
`--seed 0 --deterministic`
